Sources/ComposableArchitecture/Effect.swift

```diff
@@ -161,12 +161,19 @@ public struct Effect<Output, Failure: Error>: Publisher {
   /// - Parameter work: A closure that accepts a `Subscriber` value and returns a cancellable. When
   ///   the `Effect` is completed, the cancellable will be used to clean up any resources created
   ///   when the effect was started.
-  public static func async(
+  public static func run(
     _ work: @escaping (Effect.Subscriber<Output, Failure>) -> Cancellable
   ) -> Self {
     AnyPublisher.create(work).eraseToEffect()
   }
 
+  @available(*, deprecated, renamed: "run")
+  public static func async(
+    _ work: @escaping (Effect.Subscriber<Output, Failure>) -> Cancellable
+  ) -> Self {
+    self.run(work)
+  }
+
```

Comment:
Not sure it's necessary to deprecate, but may as well for a version or two.

Reply:
Yeah might as well moving forward. Could even create a `Deprecations.swift` in `Internal` to put all of these?